client/keys/utils.go

```diff
@@ -45,24 +45,16 @@ func NewInMemoryKeyBase() keys.Keybase { return keys.NewInMemory() }
 // NewKeyBaseFromHomeFlag initializes a keyring based on configuration. Keybase
 // options can be applied when generating this new Keybase.
 func NewKeyringFromHomeFlag(input io.Reader, opts ...keys.KeybaseOption) (keys.Keybase, error) {
```

Comment:
Whats the point of having these? The user can just call `keys.NewKeyring(keyringServiceName, keyringBackend, rootDir, input, opts...)`

Reply:
This always been a convenience function call. I never liked it cause it hides too much information IMHO. I'm glad to remove it :+1: